UnitsNet.Tests/CustomCode/CoefficientOfThermalExpansionTests.cs

```diff
@@ -30,7 +30,7 @@ public class CoefficientOfThermalExpansionTests : CoefficientOfThermalExpansionT
     {
         protected override double InverseDegreeCelsiusInOneInverseKelvin => 1.0;
 
-        protected override double InverseDegreeFahrenheitInOneInverseKelvin => 1.8;
+        protected override double InverseDegreeFahrenheitInOneInverseKelvin => 0.5555555555555556;
```

Comment:
Verified.
https://www.wolframalpha.com/input/?i=Inverse+Degree+Fahrenheit+In+One+Inverse+Kelvin